.github/workflows/workers.yml

```diff
@@ -0,0 +1,66 @@
+name: Worker builds
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+  schedule:
+    # At 23:18 every Wednesday, 2 hours before `photoview`. Details in https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule
+    - cron: '18 23 * * 3'
+
+env:
+  IS_PUSHING_IMAGES: ${{ github.event_name == 'pull_request' && github.repository == 'photoview/photoview' }}
+  DOCKER_USERNAME: viktorstrate
+  DOCKER_IMAGE: viktorstrate/workers
+  DOCKER_PASSWORD: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Regarding the build failure:
I have no idea why is it failed((
I see no difference between this one and the build workflow. Also, as I know, a secret cannot be associated with the particular workflow and accessible from all of them.
So, it should work.

As we know from the other repo, if we get the 'password required' error, it might mean that the password and username don't match. But they should here...

An alternative solution would be to create a matrix of images in the build workflow and build both images there in parallel. As far as I see from the YAML here, there should be no major issues with such implementation and all the differences can be handled by the matrix and GitHub expressions

Reply:
https://github.com/kkovaletp/photoview/blob/hw-gpu-support/.github/workflows/build.yml - here is the example of the `build.yml` with a matrix, building and pushing 2 different images to the same Docker registry

Reply:
[The login step](https://github.com/photoview/photoview/actions/runs/10794484176/job/29938871179#step:5:1) shows: <img width="637" alt="Screenshot 2024-09-10 at 21 34 16" src="https://github.com/user-attachments/assets/7efa6a9d-12fc-465d-b934-e990f9436a0c">

I read this that `DOCKER_PASSWORD` is an empty string.

Other workflows like [Docker builds](https://github.com/photoview/photoview/actions/runs/10794484156/job/29938871148?pr=1047#step:3:9) running with PR:
<img width="569" alt="Screenshot 2024-09-10 at 21 35 44" src="https://github.com/user-attachments/assets/6bfa1f22-130d-4a66-80e8-7cb5d3ded16c">

It's also an empty string. That's why I assume the secret can't be accessed with PR but only with master CI.